Building mu-avro-0.4.0.0 fails

The error:

```
Building library for mu-avro-0.4.0.0..
[1 of 4] Compiling Data.Time.Millis ( src/Data/Time/Millis.hs, dist/build/Data/Time/Millis.o )
[2 of 4] Compiling Mu.Adapter.Avro  ( src/Mu/Adapter/Avro.hs, dist/build/Mu/Adapter/Avro.o )
[3 of 4] Compiling Mu.Quasi.Avro    ( src/Mu/Quasi/Avro.hs, dist/build/Mu/Quasi/Avro.o )
[4 of 4] Compiling Mu.Quasi.Avro.Example ( src/Mu/Quasi/Avro/Example.hs, dist/build/Mu/Quasi/Avro/Example.o )

src/Mu/Quasi/Avro/Example.hs:1:1: error:
    Exception when trying to run compile-time code:
      ./test/avro/example.avdl: openFile: does not exist (No such file or directory)
    Code: avdl
            "ExampleProtocol" "ExampleService" "." "test/avro/example.avdl"
  |
1 | {-# language CPP             #-}
```

I guess the avdl file needs to be added to [the cabal file](https://github.com/higherkindness/mu-haskell/blob/f44f1f56244f0f0aa995e7aba28ebf64829974aa/adapter/avro/mu-avro.cabal#L15)

--- a/adapter/avro/mu-avro.cabal
+++ b/adapter/avro/mu-avro.cabal
@@ -1,5 +1,5 @@
 name:          mu-avro
-version:       0.4.0.0
+version:       0.4.0.1
 synopsis:      Avro serialization support for Mu microservices
 description:
   You can use @mu-avro@ to read AVRO Schema Declarations for mu-haskell
@@ -12,7 +12,7 @@ copyright:     Copyright © 2019-2020 <http://47deg.com 47 Degrees>
 category:      Network
 build-type:    Simple
 cabal-version: >=1.10
-data-files:    test/avro/*.avsc
+data-files:    test/avro/*.avsc, test/avro/*.avdl
 homepage:      https://higherkindness.io/mu-haskell/
 bug-reports:   https://github.com/higherkindness/mu-haskell/issues
 

